test/unit-test/test_lib/test_vis.py

```diff
@@ -1,49 +1,53 @@
 import os
-import sys
 import unittest
-from os import path
-from mock import patch
+
 import matplotlib.pyplot as plt
-import plotly.plotly as py
-import mock as Mock
 import numpy as np
-import networkx as nx
 import plotly.graph_objs as go
 from ddt import ddt, data, unpack
+from mock import patch
 
 from lib import vis, util
 
-sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
-current_dir = os.path.dirname(os.path.abspath(__file__))
 
+def mock_exponential_curve_func(x, a, b, c):
```

Comment:
Since these mock functions are being used only inside the test class in this module, is it better to move the functions into the VisTest class?
@Kausam and @kaushik-rohit please pitch in with the prevailing convention for this kind of mock functions.